apps/rehabstod/src/pages/CareProvider/components/CareProviderAccordion.tsx

```diff
@@ -1,4 +1,4 @@
-import { IDSIcon } from '@frontend/ids-react-ts'
+import { IDSIcon, IDSRadio } from '@frontend/ids-react-ts'
 import { ReactNode } from 'react'
 import { Radio } from '../../../components/Form/Radio'
```

Comment:
Radio can be removed, and let's remove the component completely now, since it's no longer used.